example/lib/main.dart

```diff
@@ -37,6 +37,11 @@ void main() async {
   await LMFeedCore.instance.initialize(
     lmFeedClient: lmFeedClient,
     domain: "feedsx://www.feedsx.com/",
+    config: LMFeedConfig(
+      composeConfig: const LMFeedComposeScreenConfig(
+        enablePolls: true,
```

Comment:
Let's keep default as true, so that we don't have to pass config in our master sample app ideally.